includes/wikia/DefaultSettings.php

```diff
@@ -1748,3 +1748,8 @@
  * ONLY UPDATE THROUGH WIKI FACTORY ON COMMUNITY - it's an instant global.
  */
 $wgAdDriverIncontentPlayerSlotCountries = null;
+
+/**
+ * flag for using (possibly external) service for managing a user's preferences
+ */
+$wgPreferencesUseService = true;
```

Comment:
I was imagining that we'll set this to `false` by default then tweak the config with something like:

```php
if ( ($wgCityId % 100) < 5) {
 $wgPreferencesUseService = true;
}
```